code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -740,3 +740,10 @@
 
 /mob/living/simple_animal/proc/stop_deadchat_plays()
 	stop_automated_movement = FALSE
+
+// -- LC13 THINGS --
+
+/mob/living/simple_animal/proc/CheckCombat() //Is it currently a combat gamemode? Used to check for a few interactions, like if somethings can teleport.
```

Comment:
Honestly the proc should be renamed to something more descriptive, as "CheckCombat" tells me nothing.
I know it's not your fault for naming it like that, but it would be a bonus if we renamed it to something like "IsStupidCombatMode()", you get the idea.

Reply:
I'd MUCH much rather do this on a later pr to not disrupt the current rcorp pr, so i gently do ask to wait for a later pr that changes it to IsCombatMap().